myapp/Gemfile

```diff
@@ -1,11 +1,15 @@
-source 'https://rubygems.org'
+source "https://rubygems.org"
 
-gem 'sidekiq-delay_extensions', :path => '..'
-gem 'rails'
-gem 'puma'
+gem "sidekiq-delay_extensions", path: ".."
```

Comment:
arguably in future releases we'll want to track everything in sidekiq's "myapp" except for this line